source/fundamentals/logging.txt

```diff
@@ -0,0 +1,151 @@
+
+Each message generated by a MongoDB cluster is assigned a *category*. This lets you 
+specify different log levels for different types of messages. 
+
+MongoDB uses the following categories to classify messages:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * - Category
+     - Description
+
+   * - ``MongoDB.Command``
+     - | The progress of commands run against your cluster, represented by the 
+         ``CommandStarted``, ``CommandSucceeded``, and ``CommandFailed`` events
+
+   * - ``MongoDB.SDAM``
+     - | Changes in the topology of the cluster, including the 
+         ``ClusterAddedServer``, ``ClusterRemovedServer``, ``ServerHeartbeatStarted``,
+          ``TopologyDescriptionChanged``, and ``ServerDescriptionChangedEvent`` events 
+
+   * - ``MongoDB.ServerSelection``
+     - | The decisions that determine which server to send a particular command to
+
+   * - ``MongoDB.Connection``
+     - | Changes in the cluster connection pool, including the ``PoolReady``, 
+         ``PoolClosed``, ``ConnectionCreated``, and ``ConnectionCheckout`` events 
+
+   * - ``MongoDB.Internal.*``
+     - | Prefix for all other {+driver-short+} internal components
+
+.. tip::
+ 
+   To log messages from all components, use the ``Default`` category. 
+
+Configure Log Verbosity
+-----------------------
+
+You can configure the log verbosity of each message category either in memory or in an 
+``appsettings.json`` file. 
+
+To configure the log verbosity of each message category in memory, create a 
+``Dictionary<string, string>`` where the key is ``"LogLevel:<category>" and the value 
+is the minimum log level of messages in that category. Add the dictionary to a 
+``ConfigurationBuilder`` object, then add the ``ConfigurationBuilder`` to your 
+``LoggerFactory``. 
+
+The following code sample shows how to configure a ``MongoClient`` to log two types 
+of messages: 
+
+- All messages with log level ``Error`` or higher from all categories 
+- All messages with log level ``Debug`` or higher from the SDAM category 
+
+.. code-block:: csharp
+
+   var categoriesConfiguration = new Dictionary<string, string>
+   {
+     { "LogLevel:Default", "Error" },
+     { "LogLevel:MongoDB.SDAM", "Debug" }
+   };
+
+   var config = new ConfigurationBuilder()
+     .AddInMemoryCollection(categoriesConfiguration)
+     .Build();
+
+   using var loggerFactory = LoggerFactory.Create(b =>
+   {
+     b.AddConfiguration(config);
+     b.AddSimpleConsole();
+   });
+
+   var settings = MongoClientSettings.FromConnectionString("<your connection string>");
+   settings.LoggingSettings = new LoggingSettings(loggerFactory);
+   var client = new MongoClient(settings);
+
+.. tip::
+
+   For more information on configuring log verbosity, including how to create an 
+   ``appsettings.json file``, see the 
+   `Microsoft .NET logging documentation. <https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line#configure-logging>`__
```

Comment:
+1

Reply:
I removed the mention of `appsettings.json` because it's no longer mentioned earlier on the page. Let me know if you think this Tip box needs to mention that there are other configuration methods.